gpustack/routes/update.py

```diff
@@ -0,0 +1,12 @@
+from fastapi import APIRouter
+
+from gpustack.server.update_check import get_update
+
+
+router = APIRouter()
+
+
+@router.get("/")
```

Comment:
@router.get("")

Reply:
They are equivalent but "/" is more readable